File: domein/src/main/java/be/genealogie/domein/dto/GenealogischDriehoekjeDTO.java
package be.genealogie.domein.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class GenealogischDriehoekjeDTO {
    private NatuurlijkPersoonDTO kind;
    private NatuurlijkPersoonDTO ouder1;
    private NatuurlijkPersoonDTO ouder2;
}

File: frontend/src/app/model/genealogisch-driehoekje-dto.ts
import {NatuurlijkPersoonDto} from "./natuurlijk-persoon-dto";

export class GenealogischDriehoekjeDto {
  public constructor(
    public kind: NatuurlijkPersoonDto,
    public moeder: NatuurlijkPersoonDto,
    public vader: NatuurlijkPersoonDto) {}

}
